lspSmithy/src/org/scala/abusers/lspsmithy/SmithyConverter.scala

```diff
@@ -93,11 +92,12 @@ object SmithyConverter:
             .filterNot(_.proposed)
             .filter {
               _.value match
-                case _: Int      => true
+                case _: Int      => false
                 case str: String => str.nonEmpty
             }
```

Comment:
issue: use  `.stringValue` to assert non-int (ints shouldn't be here) and add a comment linking to https://github.com/smithy-lang/smithy/issues/2626